drivers/sensor/ms5607/Kconfig

```diff
@@ -0,0 +1,57 @@
+# MS5607 pressure sensor configuration options
+
+# Copyright (c) 2019 Thomas Schmid <tom@lfence.de>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MS5607
+	bool "MS5607 pressure and temperature sensor"
+	depends on SPI
+	help
+	 Enable driver for MS5607 pressure and temperature sensor.
```

Comment:
Nit: Should be two spaces before help text instead of one.

Reply:
done